TERMS.md

```diff
@@ -0,0 +1,78 @@
+Core Cosmology Library: Terms of reference
+---------------------------------------
+
+This document lays out terms of reference for CCL developers and contributors in the wider TJP and LSST DESC context.
+
+Background
+---------------------------------------
+
+CCL was created to provide theoretical predictions of large-scale structure statistics for DESC cosmological analyses but it has also become a publicly available tool. 
+
+Scope
+---------------------------------------
+
+CCL has a commitment to support key DESC analyses. In this regard, CCL aims to
+
+- support cosmological models that are priorities for WGs (e.g., as defined by the "Beyond wCDM" taskforce);
+
+- ensure sufficient flexibility such that modifications can be implemented at a later stage as needed (e.g., models for observational systematics or changes to how the non-linear power spectrum is computed for various parts of the analysis);
+
+- support core predictions (e.g., shear correlation functions), but not necessarily every statistic that could be computed from those core predictions (e.g., COSEBIs).
+
+CCL interface support is as follows.
+
+- Public APIs are maintained in Python only (i.e., version bumping is not necessary following changes in the C interface).
+
+- The C implementation can be used for speed as needed.
+
+- Documentation should be maintained in the main README, readthedocs and the benchmarks folder.
+
+- Examples will be maintained in a separate repository.
```

Comment:
Just double-checking -- does this mean we will create a new `ccl-examples` repo or similar?

Reply:
Yes